### PR TITLE
Narration sd/sanity image safeties

### PR DESCRIPTION
**Pull request by narration-sd**
Added simple missing-image safety to both apps, in src/components/SanityImage.astro

### CODE REVIEW

File: apps/example-ssr/src/pages/posts/[slug].astro [hunk 1]
```diff
@@ -3,6 +3,8 @@ import Layout from "../../layouts/Layout.astro";
 import { useSanityClient } from "@sanity/astro";
 import PortableText from "../../components/PortableText.astro";
 import SanityImage from "../../components/SanityImage.astro";
+// import '@astrojs/prism'
+// import 'prismjs/components/prism-markup-templating'
```
Comment (kmelve) [inline apps/example-ssr/src/pages/posts/[slug].astro:7]: remove these?

**Review by kmelve — CHANGES_REQUESTED** [review]
Thanks for the PR! We can merge this if you could look at the comment 🙇

File: apps/example-ssr/src/pages/posts/[slug].astro (hunk 1 above)
Comment (narration-sd) [reply apps/example-ssr/src/pages/posts/[slug].astro:7]: Yes indeed, sorry -- those were leftovers from chasing an apparent tree-shaking problem in recent Astro w/Netlify....

**Comment by narration-sd** [comment]
Closing, as current commit has a nicer method, covering the original issue

**Pull request closed without merging**